index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Selouse</title>
+
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css"
+        integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="style.css" />
+</head>
+
+<body>
+
+    <div id="search">
+        <input type="text" placeholder="Address" id="addr-input">
+        <input type="number" class="min price" placeholder="Min. Price" id="min-p-input">
+        <input type="number" class="max price" placeholder="Max Price" id="max-p-input">
+        <input type="number" class="min room" placeholder="Min. Rooms" id="min-r-input">
+        <input type="number" class="max room" placeholder="Max Rooms" id="max-r-input">
+
+        <div class="check-group">
+            <input class="form-radio" type="checkbox" name="immediate" id="immed-y">
+            <label for="immediate">Immediate</label>
+        </div>
+        <div class="check-group">
+            <input class="form-radio" type="checkbox" name="parking" id="park-y">
+            <label for="parking">Parking</label>
+        </div>
+
+        <button>Find Apartments</button>
+    </div>
+
+    <hr>
+
+    <div id="results"></div>
+
+    <!-- Your Code Here -->
+
+    <!-- You should start doing handlebar stuff here -->
+    <script id="apartment-template" type="text/x-handlebars-template">
```

Comment:
Make sure you indent your handlebars correctly according to HTML indent rules. It makes the code really noisy when you don't format it.